Jellyfin.Server.Implementations/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -11,17 +17,77 @@ namespace Jellyfin.Server.Implementations.Extensions;
 /// </summary>
 public static class ServiceCollectionExtensions
 {
+    private static IEnumerable<Type> DatabaseProviderTypes()
+    {
+        yield return typeof(SqliteDatabaseProvider);
+    }
+
+    private static IDictionary<string, JellyfinDbProviderFactory> GetSupportedDbProviders()
+    {
+        var items = new Dictionary<string, JellyfinDbProviderFactory>();
+        foreach (var providerType in DatabaseProviderTypes())
+        {
+            var keyAttribute = providerType.GetCustomAttribute<JellyfinDatabaseProviderKeyAttribute>();
+            if (keyAttribute is null || string.IsNullOrWhiteSpace(keyAttribute.DatabaseProviderKey))
+            {
+                continue;
+            }
+
+            var provider = providerType;
+            items[keyAttribute.DatabaseProviderKey.ToUpperInvariant()] = (services) => (IJellyfinDatabaseProvider)ActivatorUtilities.CreateInstance(services, providerType);
```

Comment:
Pretty minor thing, but you could have used StringComparer.OrdinalIgnoreCase in the dictionary ctor and avoided ToUpperInvariant